docs/sources/operators-guide/configuring/configuring-object-storage-backend.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Configuring Grafana Mimir object storage backend"
+menuTitle: "Configuring object storage"
```

Comment:
```suggestion
menuTitle: "Configure object storage"
```

Reply:
Done in https://github.com/grafana/mimir/pull/2347/commits/7f7abdc93b27a7ea8bb85096c38437a6600e04a1